### ISSUE TITLE
`patterns()` doesn't exist on django 1.11 but is necessary for django-machina stable

### ISSUE THREAD

**Issue by Melkor333** [ellmetha/django-machina#153]
I want to add the stable version of `django-machina` to an existing app running on django 1.11. 

The manual says for the stable version to use `urlpatterns = patterns(` in the url configuration but `patterns` doesn't exist in django 1.11.
```
from machina.app import board

urlpatterns = patterns(
    # [...]
    # Apps
    url(r'^forum/', include(board.urls)),
)
```

And without patterns I get the error `sre_constants.error: unknown extension ?P1 at position 9`.

Is there some workaround for this or did I miss something?

**Comment by sneezyyzeens** [comment]
You can use the newer path() instead of url() (don't forget to import **include** from django.urls at the top of the file).

`path('forum/', include(board.urls), name='forum'),`

**Comment by Melkor333** [comment]
Thanks a lot for your answer!
Sadly I can't use `path()`, because like `patterns()` this does only exist in Django version 2.0+ as you can see here:
https://docs.djangoproject.com/en/2.1/ref/urls/

Is there a way to not need any Django 2.0+ Features?
If not, can you fix the documentation for version 7.0 to not mention django 1.11 as this would be misleading?

**Comment by ellmetha** [comment]
Django-machina now requires Django 2.0+. Closing this issue. 😉
